.travis.yml

```diff
@@ -2,8 +2,12 @@ language: php
 
 sudo: false
 
-php:
-  - 5.5
+matrix:
+  include:
+    - php: 5.5
+      env: WITH_LOWEST=true
+    - php: 5.5
+      env: WITH_HIGHEST=true
```

Comment:
Sure can, but the idea is that it shows right here https://travis-ci.org/codeclimate/php-test-reporter/builds/190697849:

![screen shot 2017-01-10 at 19 45 36](https://cloud.githubusercontent.com/assets/605483/21819797/72f3aeb8-d76d-11e6-93d6-33aa3260726c.png)

Happy to remove it, though, if you insist!